### ISSUE TITLE
Learning examples consistently have objects facing the learner, leading to spurious learning

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#696]
We learn the fact that an object faces the learner is part of the verb semantics in almost all cases because it appears in all our training examples.

Example:

[slot1_jumps.pdf](https://github.com/isi-vista/adam/files/4371837/slot1_jumps.pdf)

**Comment by LichMaster98** [comment]
@gabbard Could this be the result of this hack: https://github.com/isi-vista/adam/blob/master/adam/situation/templates/phase1_templates.py#L690-L721 ?

**Comment by gabbard** [comment]
@LichMaster98 : yep, I think that's most likely it.  I'll mark this as low priority and leave it aside for now.
